applications/CompressiblePotentialFlowApplication/python_scripts/define_wake_process_2d.py

```diff
@@ -32,7 +32,13 @@ def __init__(self, Model, settings):
         self.epsilon = settings["epsilon"].GetDouble()
 
         self.fluid_model_part = self.body_model_part.GetRootModelPart()
-        self.trailing_edge_model_part = self.fluid_model_part.CreateSubModelPart("trailing_edge_model_part")
+        if not self.fluid_model_part.HasSubModelPart("trailing_edge_model_part"):
+            self.trailing_edge_model_part = self.fluid_model_part.CreateSubModelPart("trailing_edge_model_part")
+        else: self.trailing_edge_model_part = self.fluid_model_part.GetSubModelPart("trailing_edge_model_part")
+
+        if not self.fluid_model_part.HasSubModelPart("wake__elements"):
+            self.wake_elements_sub_modelpart = self.fluid_model_part.CreateSubModelPart("wake__elements")
```

Comment:
I would call this model part wake_modelpart
```suggestion
            self.wake_modelpart = self.fluid_model_part.CreateSubModelPart("wake__modelpart")
```